api/PclusterApiHandler.py

```diff
@@ -360,7 +357,7 @@ def sacct():
             if not isinstance(_price_guess, tuple):
                 price_guess = _price_guess
         except Exception as e:
-            print(e)
+            pass
```

Comment:
thought: shall we log the exception in this case?

Reply:
We could probably raise the exception and let our handlers deal with that (log and return the error)
Yes I'll do that